playbooks/azure/openshift-cluster/provisioning_vars.yml.example

```diff
@@ -0,0 +1,76 @@
+---
+# Variables that are commented in this file are optional; uncommented variables
+# are mandatory.
+
+# Default values for each variable are provided, as applicable.
+# Example values for mandatory variables are provided as a comment at the end
+# of the line.
+
+# ------------------------ #
+# Common/Cluster Variables #
+# ------------------------ #
+# Variables in this section affect all areas of the cluster
+
+# Deployment type must be specified.
+openshift_deployment_type: # 'origin' or 'openshift-enterprise'
+
+# specify a clusterid
+# This value is also used as the default value for many other components.
+openshift_azure_clusterid:  # e.g test
+
+# azure region
+# This value will instruct the plays where all items should be created.
+# Multi-region deployments are not supported using these plays at this time.
+openshift_azure_region:  # e.g. eastus
+
+# -------- #
+# ssh keys #
+# -------- #
+
+# Specify the key pair name here to connect to the provisioned instances.
+#openshift_azure_vm_admin_username: openshift
+openshift_azure_vm_ssh_public_keys:
+- path: "/home/{{ openshift_azure_vm_admin_username }}/.ssh/authorized_keys"
+  key_data: "ssh-rsa AAAAB3... {{ openshift_azure_vm_admin_username }}_key"
+
+
+# image settings
+openshift_azure_vm_base_image_name: rhel7.4_base
+openshift_azure_vm_base_image:
+  name: "{{ openshift_azure_vm_base_image_name }}"
+  resource_group: "{{ openshift_azure_resource_group_name }}"
+
+openshift_azure_vm_storage_container: "{{ openshift_azure_clusterid }}"
+openshift_azure_storage_account_name: "openshift{{ openshift_azure_clusterid }}"
+
+# --------- #
+# AMI Build #
+# --------- #
+# Variables in this section apply to building a node AMI for use in your
+# openshift cluster.
+# openshift-ansible will perform the container runtime storage setup when specified
+# The current storage setup with require a drive if using a separate storage device
+# for the container runtime.
+container_runtime_docker_storage_type: overlay2
+container_runtime_docker_storage_setup_device: /dev/xvdb
```

Comment:
is this actually used or is it aws-specific? can this file generally be made a little clearer?

Reply:
The docker drive requires a drive for docker-storage-setup.  It should be passed into the container_runtime role.

Reply:
/dev/xvdb can't be doing anything for us because it's amazon-specific (xen hypervisor). On azure it'd be /dev/sdb?